frame/support/test/tests/pallet_ui/storage_value_unexpected_named_generic.stderr

```diff
@@ -0,0 +1,5 @@
+error: Invalid `StorageValue`, cannot find `Value` generic
+  --> $DIR/storage_value_unexpected_named_generic.rs:19:28
+   |
+19 |     type Foo<T> = StorageValue<P = u32>;
```

Comment:
Can we point to `P` and say that it was unexpected?

Reply:
indeed, the error message is now:
```
error: Invalid pallet::storage, Unexpected generic `P` for `StorageValue`. `StorageValue` expect generics `Value`, and optional generics `QueryKind`, `OnEmpty`.
  --> $DIR/storage_value_unexpected_named_generic.rs:19:29
   |
19 |     type Foo<T> = StorageValue<P = u32>;
   |                                ^

error: Invalid pallet::storage, cannot find `Value` generic, required for `StorageValue`.
  --> $DIR/storage_value_unexpected_named_generic.rs:19:28
   |
19 |     type Foo<T> = StorageValue<P = u32>;
   |                               ^
```
So it contains all information